tests/conversion/advanced-converters.js

```diff
@@ -45,7 +45,7 @@ let modelDoc, modelRoot, viewRoot, mapper, writer, modelDispatcher, viewDispatch
 
 beforeEach( () => {
 	modelDoc = new ModelDocument();
-	modelRoot = modelDoc.createRoot( 'root' );
+	modelRoot = modelDoc.createRoot( '$root', 'root' );
```

Comment:
In tests that use `doc.createRoot( '$root', 'root' );` we should use simpler version `doc.createRoot()` and eventually fix places where `root` name is used to use `main`. Is that correct @pjasiun?